tests/integration/ledger/README.md

```diff
@@ -0,0 +1,13 @@
+To run the test a ledger device must be connected and the Ethereum application running on it.
+
+Install ledger requirement: 
+```sh
+# in the project root:
+pip install -e .[ledger]
+```
+
+Run the tests with:
```

Comment:
Probably need to set up the ledger somehow first right? From the other comments, maybe we need a shared seed for testing.